apps/judicial-system/backend/src/app/modules/subpoena/subpoena.service.ts

```diff
@@ -66,6 +73,10 @@ export const include: Includeable[] = [
         model: Institution,
         as: 'court',
       },
+      {
+        model: CourtSession,
+        as: 'courtSessions',
+      },
```

Comment:
We need the court sessions to determine if a court documents should be added for subpoena service certificates.